include/quicr/detail/time_queue.h

```diff
@@ -24,26 +24,16 @@
 #include <array>
 #include <atomic>
 #include <chrono>
-#include <iostream>
 #include <memory>
 #include <mutex>
 #include <sys/select.h>
 #include <thread>
 #include <type_traits>
 #include <vector>
 
-namespace quicr {
-
-    /**
-     * Interface for services that calculate ticks.
-     */
-    struct TickService
-    {
-        using TickType = size_t;
-        using DurationType = std::chrono::microseconds;
+#include "quicr/detail/tick_service.h"
```

Comment:
This include could just be
```suggestion
#include "tick_service.h"
```
since they are in the same directory.